config/i18n/locales/german/translations.json

```diff
@@ -0,0 +1,97 @@
+{
+  "buttons": {
+    "forum": "Forum",
+    "donate": "Spende",
+    "load-more-articles": "Weitere Artikel laden"
+  },
+  "search": {
+    "label": "Suche",
+    "accessible-name": "Sende deine Suchanfrage ab",
+    "placeholder": "Mehr als 8.000 Tutorials durchsuchen",
+    "see-results": "Zeige alle Ergebnisse für {{ searchQuery }}",
+    "no-tutorials": "Keine Tutorials gefunden"
+  },
+  "banner": {
+    "default": "Programmieren lernen - {{ <0> }}kostenloser 3.000-Stunden-Lehrplan{{ </0> }}",
+    "authenticated": "Unterstütze unsere gemeinnützige Organisation und unsere Mission. {{ <0> }}Spende an freeCodeCamp.org{{ </0> }}.",
+    "authenticated-donor": "Vielen Dank, dass du das freeCodeCamp durch deine Spenden unterstützt{{ <0> }}."
```

Comment:
The English version of this string is the following:

```
"authenticated-donor": "Thank you for supporting freeCodeCamp through {{ <0> }}your donations{{ </0> }}."
```

But the German version doesn't have wrappers (`{{ <0> }} and {{ </0> }}`) around any text:

```
    "authenticated-donor": "Vielen Dank, dass du das freeCodeCamp durch deine Spenden unterstützt{{ <0> }}."
```

Is it possible to wrap some text in those like with the English version of the string?

Reply:
Excuse me.

Here is a hopefully correct suggestion.

"authenticated-donor": "Vielen Dank dafür, dass du freeCodeCamp durch {{ <0> }}deine Spenden{{ </0> }} unterstützt."

Thanks for your help